naucse/views.py

```diff
@@ -568,8 +577,17 @@ def course_page(course, lesson, page, solution=None):
                     title = '{}: {}'.format(course.title, page.title)
 
                     try:
-                        prev_link, session_link, next_link = course.get_footer_links(lesson.slug, page_slug,
-                                                                                     request_url=request.path)
+                        footer_links = course.get_footer_links(
+                            lesson.slug,
+                            page_slug,
+                            request_url=request.path,
+                        )
+                        for link in footer_links:
+                            _prefix = f"/{self.slug}/"
```

Comment:
I think this is supposed to be ``course.slug``? ``self`` definitely does not exist here.